lib/archethic/account.ex

```diff
@@ -27,6 +19,8 @@ defmodule Archethic.Account do
   """
   @spec get_balance(Crypto.versioned_hash()) :: balance()
   def get_balance(address) when is_binary(address) do
+    # TODO
```

Comment:
I don't get this comment, I don't think this is relevant to keep it

Reply:
my bad, removed. rebase